Makefile

```diff
@@ -200,7 +200,7 @@ image-push-scorecard-proxy-multiarch:
 test: test-unit ## Run the tests
 
 test-markdown:
-	./hack/ci/marker -e website
+	./hack/ci/marker --root=doc
```

Comment:
Don't we also need to check files in the root (e.g. CHANGELOG.md, README.md, etc.)?

Reply:
The marker start to raise an error #2741 when the link is not broken. 
The master CI is broken as we are unable to merge the PR's.
Also, will not it be removed soon?

Reply:
For that error, it looks like coreos.com uses TLS 1.3, so `marker` is likely out-of-date.

Another related improvement: perhaps we can run marker against changed markdown files. That will solve the above issue for PRs that don't change the README (most of them).

```
#!/usr/bin/env bash

set -e

source ./hack/lib/common.sh

header_text "Running markdown link checker against changed markdown files"

COMMIT_RANGE="$TRAVIS_COMMIT_RANGE"
if [ -z "$COMMIT_RANGE" ] || ! git rev-list --quiet $COMMIT_RANGE; then
	# Assume comparing current branch to master if we don't have $TRAVIS_COMMIT_RANGE
	COMMIT_RANGE="master"
fi

find . -name '*.md' > .marker.tmp.all
git diff --name-only $COMMIT_RANGE | grep '\.md$' | sed -e 's/^/.\//' > .marker.tmp.changed
excludes=$(grep -vxF -f .marker.tmp.changed .marker.tmp.all | sed -e 's/^/--exclude /')
rm .marker.tmp.*

./hack/ci/marker $excludes
```

It _won't_ solve the issues with CHANGELOG.md errors with "429 Too Many Requests". We might need to find a different markdown link checker that can be rate-limited.